FireFlyRenderEngine/main.cpp

```diff
@@ -1,15 +1,24 @@
 #pragma once
 #include <iostream>
-
-#include "../GLFWViewer/GLFWViewer.h"
+#include "Scene/Scene.h"
+#include "GLFWViewer/GLFWViewer.h"
+#include "../glm-0.9.9.7/vec3.hpp"
```

Comment:
we don't need to use the full: "glm-0.9.9.7" anymore, but this is trivial and can be fixed in the next changes